### DISCUSSION TITLE
Usability of Folder View on Mobile Device

### DISCUSSION THREAD

**Discussion by gizmo-naut** [photoprism/photoprism#3619] [Q&A] [answered]
Hi All,
I use photoprism to have a web front-end for my private picture collection, which is well organized in a folder structure. The folders view works for me very well on the desktop. Here I can navigate the images in full screen with the left and right keyboard buttons. On the mobile device however, I find the folder view unusable. When I click on an image it shows me all kind of metadata. I would wish for a full screen view of the image I selected and the possibility to easily swipe to the next image with the finger. Just like it works in the albums section. Is there an easy way to adjust this behaviour with a configuration switch?

regards,
gizmo-naut

**Comment by graciousgrey** [comment, marked as answer]
Hey :)

Are you referring to the [folder view](https://docs.photoprism.app/user-guide/organize/folders/) or the [file browser](https://docs.photoprism.app/user-guide/library/files/) ?

If you click on images in the folder view, the photos will be opened in full screen mode (on desktop and mobile). If you click on files in the file browser, the photo edit dialog will open (on desktop and mobile).

An option to display folders hierarchically in the folder view is planned, but not yet implemented.

**Reply by gizmo-naut** [reply]
Thanks for the hint. Indeed, I mixed up the section on mobile... my bad!
